src/app/body/milestone-details/milestone-details.component.html

```diff
@@ -130,7 +130,7 @@
             <span class="title mt-2">Task Tracker</span>
           </div>
         </div>
-        <div class="process-wrapper" *ngIf="milestoneService.taskData.length > 0">
+        <div class="process-wrapper" *ngIf="milestoneService.taskData.length > 0 && !milestoneCompleted">
```

Comment:
The graph should not be modified with the status of the Milestone 
Instead it the position of the TaskIds in the graph should be altered and the graph should one and only be modified when the task is complete.